test/float/squareroot.c

```diff
@@ -0,0 +1,15 @@
+#include <stdio.h>
```

Comment:
Create a failing version of this benchmark. In general, we always aim to create both a correct and a failing version.